build-plugins/build-support/build.gradle.kts

```diff
@@ -21,9 +22,20 @@ dependencies {
     runtimeOnly(project(":ktlint-rules"))
     implementation(libs.nexusPublishPlugin)
     compileOnly(gradleApi())
+    implementation("aws.sdk.kotlin:s3:1.+")
```

Comment:
Should we pin on 1.1.+ to avoid catching breaking changes via minor version bump?